planning/behavior_path_planner/src/turn_signal_decider.cpp

```diff
@@ -65,28 +65,48 @@ std::pair<TurnIndicatorsCommand, double> TurnSignalDecider::getIntersectionTurnS
 
   // Get nearest intersection and decide turn signal
   double accumulated_distance = 0;
+  double turn_distance = intersection_search_distance_;
```

Comment:
`turn_distance` should be defined in this scope, not outside this `for` loop.

Reply:
Fixed the above.